workflow/source.go

```diff
@@ -24,6 +26,14 @@ type Document struct {
 	Auth     *Auth  `yaml:",inline"`
 }
 
+type SpeakeasyRegistryDocument struct {
+	OrgSlug       string
+	WorkspaceSlug string
+	NamespaceName string
```

Comment:
We're shooting ourselves in the foot if we chose to break out a namespace id at every place in our code. The reason we defined namespaces is in order to use that concept in code. In other words, most code should not be aware of what comprises a namespace id. Ideally all consumer code talking to speakeasy registry should communicate with a "namespace ID" or "a workspace ID and namespace name".